bot/converters.py

```diff
@@ -568,5 +499,4 @@ async def convert(self, ctx: Context, arg: str) -> t.Optional[dict]:
 
 
 Expiry = t.Union[Duration, ISODateTime]
-FetchedMember = t.Union[discord.Member, FetchedUser]
-UserMention = partial(_snowflake_from_regex, RE_USER_MENTION)
```

Comment:
Are `_snowflake_from_regex` or `RE_USER_MENTION` used anywhere else? If not, delete them too.

Reply:
`_snowflake_from_regex` isn't but it seems `RE_USER_MENTION` is used in `UserMentionOrID` here https://github.com/python-discord/bot/blob/main/bot/converters.py#L475

Reply:
`RE_USER_MENTION` is still used but not `_snowflake_from_regex` so will remove that :+1: